sentry_sdk/crons/decorator.py

```diff
@@ -1,19 +1,19 @@
-from functools import wraps
 import sys
 
 from sentry_sdk._compat import reraise
 from sentry_sdk._types import TYPE_CHECKING
 from sentry_sdk.crons import capture_checkin
 from sentry_sdk.crons.consts import MonitorStatus
 from sentry_sdk.utils import now
-
+from sentry_sdk._compat import contextmanager
```

Comment:
Let's add this to the existing `from sentry_sdk._compat...` import on line 3.

Reply:
Done -- But, I wonder if this is something we could (or should) automate by using a pre-commit hook or some other automatic code formatter

Reply:
That'd be something worth looking into. We could use https://pycqa.github.io/isort/ or something similar -- though I don't know how it'd deal with our conditional py2/py3/type checking imports, we'd need to check that.